ssh-keygentest.py

```diff
@@ -0,0 +1,14 @@
+import unittest
+from .testutils import system
+
+# This is a test case for ssh-keygen command 
```

Comment:
Change this commented out statement to docstring.
Also add this in the test instead of here so that it will be displayed when running tests.
Docstring ref: http://www.pythonforbeginners.com/basics/python-docstrings